client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -33,15 +33,16 @@ export default React.createClass( {
 
 	render() {
 		const { site, redirect, linkDisplay } = this.props;
-		const buttonElement = linkDisplay ? 'button' : Button;
```

Comment:
This prop defaults to `true` and I've found at least one component (`client/my-sites/plugins/plugin-activate-toggle`) which does not override it. I think we need to preserve the existing behavior, else there may be regressions where it's intended to be rendered as a link?